package.json

```diff
@@ -225,11 +224,13 @@
     "typedoc": "^0.20.12",
     "typescript": "^4.9.4",
     "ui-box": "^5.4.0",
+    "unist-util-visit-esm": "npm:unist-util-visit@^4.1.2",
     "vite": "^4.0.1"
   },
   "resolutions": {
     "csstype": "3.0.11",
-    "playwright": "1.28.1"
+    "playwright": "1.28.1",
+    "@types/react": "^17.0.31"
```

Comment:
Question: why is this resolution necessary?

Reply:
The mdx library was allowing types v18 to get installed. Setting a resolution for now to avoid TS errors until we upgrade to 18.